src/MuTalk-CI-Tests/MTCIMarkdownExporterTest.class.st

```diff
@@ -0,0 +1,31 @@
+Class {
+	#name : 'MTCIMarkdownExporterTest',
+	#superclass : 'TestCase',
+	#category : 'MuTalk-CI-Tests',
+	#package : 'MuTalk-CI-Tests'
+}
+
+{ #category : 'as yet unclassified' }
+MTCIMarkdownExporterTest >> testMarkdownExporter [
+
+	| analysis md moreInfo |
+	analysis := MTAnalysis new
+		            testClasses: { SmallBankTest };
+		            classesToMutate: { SmallBank }.
```

Comment:
I wonder if it's not because they need [SmallBank](https://github.com/mabdi/smalltalk-SmallBank) to run.
Should I add a check if they are on the image and if not, it installs it ?